Source/Scene/MetadataClassProperty.js

```diff
@@ -149,7 +141,6 @@ Object.defineProperties(MetadataClassProperty.prototype, {
    * @memberof MetadataClassProperty.prototype
    * @type {MetadataType}
    * @readonly
-   * @private
    */
   valueType: {
```

Comment:
`valueType` should stay `@private` since it's an implementation detail for creating metadata tables and not part of the property definition.